lib/ex_unit_properties.ex

```diff
@@ -471,4 +471,28 @@ defmodule ExUnitProperties do
         {clauses, options}
     end
   end
+
+  @doc """
+  Picks a random element generated by the `StreamData` generator `data`.
+
+  This function uses the current ExUnit seed to generate a random term from `data`. The generation
+  size (see the "Generation size" section in the documentation for `StreamData`) is chosen at
+  random between in `1..100`. If you want finer control over the generation size, you can use
+  functions like `StreamData.resize/2` to resize `data` or `StreamData.scale/2` to scale the
+  generation size.
+
+  ## Examples
+
+      ExUnitProperties.pick(StreamData.integer())
+      #=> -21
+
+  """
+  @spec pick(StreamData.t(a)) :: a when a: term()
+  def pick(data) do
+    seed = ExUnit.configuration()[:seed]
```

Comment:
Let's use the new seed in ExUnit v1.6?

Reply:
You mean the seed in the pdict? I guess that would make more sense?